sky/backends/cloud_vm_ray_backend.py

```diff
@@ -764,6 +764,11 @@ def _update_blocklist_on_gcp_error(
                 logger.warning(f'Got \'resource not found\' in {zone.name}.')
                 self._blocked_resources.add(
                     launchable_resources.copy(zone=zone.name))
+            elif 'rsync: command not found' in stderr:
+                with ux_utils.print_exception_no_traceback():
+                    raise RuntimeError(
+                        'rsync is not installed on the specific image. '
+                        'Please install rsync and try again.')
```

Comment:
style nit: we are using the same error message across multiple clouds, should be better to have a constant variable for the message.

Reply:
```suggestion
                        '`rsync` command is not found in the specified image. '
                        'Please use an image with rsync installed.')
```